### PR TITLE
Fix game spec

### PR DESCRIPTION
**Pull request by addod19**

### CODE REVIEW

**Comment by addod19** [comment]
resolved merge conflict

File: spec/game_spec.rb [hunk 1]
```diff
@@ -4,23 +4,28 @@
...
 
-  tie = Board.new(
+  win = Board.new({
+      1 => '', 2 => '', 3 => 'O',
+      4 => '', 5 => '', 6 => 'O',
+      7 => '', 8 => '', 9 => 'O'
+    }
+
+
+  tie = Board.new({
```
Comment (Oghenebrume50) [inline spec/game_spec.rb]: use let to declare these shared variables, the test wouldn't run like this

File: spec/board_spec.rb [hunk 2]
```diff
@@ -4,159 +4,59 @@
...
+      "
+      |          |
+      #{@grid[0]} | #{@grid[1]} | #{@grid[2]}
+      -----------
+      |          |
+      #{@grid[3]} | #{@grid[4]} | #{@grid[5]}
+      -----------
+      |          |
+      #{@grid[6]} | #{@grid[7]} | #{@grid[8]}
+      "
```
Comment (Oghenebrume50) [inline spec/board_spec.rb]: This isn't really necessary though, you could check for the datatype our specific content, but creating a replica of something just to test if the replica is the same with the original, isn't much of a test

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Hi Team good start to this test, I was able to run it with a little tweak :smile: , please see the below comments for required changes 

-There should be positive and negative test cases for all test, that is if you test for a truthy outcome, you should test for a falsy outcome too.
Stickler is not working on this repo yet, please add it.

make changes and request for  a re-review :+1:

**Comment by Ispirett** [comment]
**Hey, guys, you need to use the correct git-flow please make a pull request that merges to the development branch.
resubmit for a review**

**Comment by addod19** [comment]
closed

**Pull request closed without merging**